src/sentry/templates/sentry/js-sdk-loader.ts

```diff
@@ -126,6 +114,32 @@ declare const __LOADER__IS_LAZY__: any;
     _currentScriptTag.parentNode!.insertBefore(_newScriptTag, _currentScriptTag);
   }
 
+  // We want to ensure to only add default integrations if they haven't been added by the user.
+  function setupDefaultIntegrations(config: any, SDK: any) {
```

Comment:
not a blocker but isn't there a better type that we could use here?

Reply:
IMHO We'd need to port all the types from JS-SDK into here to make this meaningful. It may be worthwhile to generally add more types here, but we'd need to start at the top there - I'd keep this for a follow up PR :)